### DISCUSSION TITLE
"For Dummies" Installation and Usage for Windows Developers

### DISCUSSION THREAD

**Discussion by tscilipoti** [git-quick-stats/git-quick-stats#177] [Q&A] [answered]
I am not Unix literate. Not interested in working with the source code. Just want to collect some stats on my C#-based repos. Not finding the instructions consumable. Does anyone have a "for dummies" version of how to get the tool installed and working? I have cloned the repo locally but don't know where to start from there. Entering 'git-quick-stats' on a command line at the root of the repo directory just results in "'_git-quick-stats' is not recognized as an internal or external command, operable program or batch file._" Thanks in advance.

**Comment by tomice** [comment, marked as answer]
Hi there! 

`git-quick-stats` requires a POSIX-like environment to run. For Windows users, we recommend using [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) (Windows Subsystem for Linux). Alternatively, you can use [Cygwin](https://www.cygwin.com/), or possibly [MSYS2](https://www.msys2.org/), though the latter is untested.

If you prefer not to install any of the above, or if you don’t have access to them, consider using an alternative project we also support: [git-py-stats](https://github.com/git-quick-stats/git-py-stats). This Python-based tool generates stats and requires Python 3 (which can be verified with `python --version` in your terminal).

Currently, these are the best options. A full native Windows port would likely require porting the code to PowerShell or using a compiled language like C# or C++, and neither of those are currently in the works.
